example_test.go

```diff
@@ -0,0 +1,109 @@
+package tfe
+
+import (
+	"bytes"
+	"context"
+	"log"
+
+	slug "github.com/hashicorp/go-slug"
+)
+
+func ExampleConfigurationVersionsUploadRaw() {
```

Comment:
Thanks for this feedback! The `ExampleFunc()` idiom is so that examples[ show up in the GoDocs](https://go.dev/blog/examples). What I'll do is have both examples in `examples/` (if the user prefers to view them through Github) as well as in `example_test.go` if the user prefers to look at the GoDocs. I will update our existing examples as well.

Reply:
No I've decided the examples won't be run as part of our test suite for the time being. Examples can be optionally executed and will only do so if the example function contains a `// Output: <some value>` comment at the end of the body.